core/chains/evm/txmgr/evm_tx_store.go

```diff
@@ -1172,7 +1215,9 @@ ORDER BY nonce ASC
 		if err = orm.LoadTxesAttempts(ctx, etxs); err != nil {
 			return pkgerrors.Wrap(err, "FindTransactionsConfirmedInBlockRange failed to load evm.tx_attempts")
 		}
-		err = orm.loadEthTxesAttemptsReceipts(ctx, etxs)
+
+		// retrieve tx with attempts which contain attempts and partial receipts values for optimization purpose
```

Comment:
nit: also mentions attempts twice here
```suggestion
		// retrieve tx with attempts and partial receipt values for optimization purpose
```